src/utils/tests/helpers.ts

```diff
@@ -148,7 +151,7 @@ export const getNetwork = (
     lndNodes: 2,
     clightningNodes: 1,
     eclairNodes: 1,
-    bitcoindNodes: 1,
+    bitcoindNodes: 2,
```

Comment:
It's not clear to me why you needed to increase the number of bitcoin nodes for the tests. Why was this necessary?

Reply:
Do we need this change?